ETC.tsv

```diff
@@ -7648,9 +7648,9 @@ ETC_20150317_007648	$Awaken the memories of the Sleeping Owl Sculptures
 ETC_20150317_007649	{memo X}Mercenary of valley
 ETC_20150317_007650	{memo X}Special Device of Valley
 ETC_20150317_007651	{memo X}Disappeared Research Grop (3)
-ETC_20150317_007652	$Remove the West Barrier Device
-ETC_20150317_007653	$Remove the East Barrier Device
-ETC_20150317_007654	$Remove the North Barrier Device
+ETC_20150317_007652	$Remove the Western Barrier
+ETC_20150317_007653	$Remove the Eastern Barrier
+ETC_20150317_007654	$Remove the Northern Barrier
```

Comment:
These appear to be devices in Akmen Ridge right? The barrier that traps Airene is powered by these devices.